pallets/kensetsu/src/lib.rs

```diff
@@ -470,8 +460,6 @@ pub mod pallet {
         CDPUnsafe,
         /// Too many CDPs per user
         CDPLimitPerUser,
-        /// Risk management team size exceeded
-        TooManyManagers,
```

Comment:
Do we already have Kensetsu on prod?
If yes, the shifting of error id could corrupt the history of indexers.

@vovac12 what do you think? Can we just remove it or leave as "unused error"?

Reply:
No, we do not have Kensetsu on prod